new/release-cadence.md

```diff
@@ -0,0 +1,49 @@
+---
+RFC: unassigned
+Title: Chef Release Cadence
+Author: Tim Smith <tsmith@chef.io>
+Author: Noah Kantrowitz <noah@coderanger.net>
+Status: Draft
+Type: Informational
+---
+
+# Chef Release Cadence
+
+Chef follows [Semantic Versioning](https://semver.org/) for releases. Major
+versions (eg. 11.x -> 12.x) will include backwards-incompatible changes, minor
+versions (eg 12.1 -> 12.2) will include new features and bug fixes but will be
+backwards-compatible to the best of our ability. Patch versions are governed
+by [RFC 47](rfc047-release-process.md).
+
+Chef feature releases are promoted to the stable channel once per month. It is
+expected that this occur during the second week of the month unless
+circumstances intervene. Additional patch releases for a given feature release
+may be promoted if critical issues are found.
+
+ChefDK is released once per month. It is expected that this occur during the
+fourth week of the month unless circumstances intervene.
+
+Both Chef and ChefDK will prepare a release candidate in the week prior to the
```

Comment:
This is already covered in RFC-047. Can we delete the language here so it's not covered in two places?